2021-10-25 17:35:50.933 | INFO     |  5713 | mycroft.messagebus.load_config:load_message_bus_config:33 | Loading message bus configs
2021-10-25 17:35:50.948 | INFO     |  5713 | mycroft_bus_client.client.client | Connected
2021-10-25 17:35:50.951 | INFO     |  5713 | mycroft.util.process_utils:start_message_bus_client:155 | Connected to messagebus
2021-10-25 17:35:50.963 | INFO     |  5713 | mycroft.skills.msm_wrapper:create_msm:96 | Acquiring lock to instantiate MSM
2021-10-25 17:35:52.221 | INFO     |  5713 | msm.mycroft_skills_manager | building SkillEntry objects for all skills
2021-10-25 17:35:53.108 | INFO     |  5713 | mycroft.skills.msm_wrapper:create_msm:112 | Releasing MSM instantiation lock.
2021-10-25 17:35:53.109 | INFO     |  5713 | mycroft.skills.skill_updater:_log_next_download_time:267 | Next scheduled skill update: 2021-10-25 17:35:52.108937
2021-10-25 17:35:53.109 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-pairing
2021-10-25 17:35:53.831 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:35:53.851 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-pairing
2021-10-25 17:35:53.851 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:35:53.853 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-pairing.mycroftai
2021-10-25 17:35:53.861 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-pairing.mycroftai/settings.json
2021-10-25 17:35:53.871 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-pairing.mycroftai
2021-10-25 17:35:53.872 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-pairing.mycroftai/settings.json
2021-10-25 17:35:53.877 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-pairing.mycroftai loaded successfully
2021-10-25 17:35:53.878 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-volume
2021-10-25 17:35:54.465 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:35:54.498 | INFO     |  5713 | msm.skill_entry | Installing requirements.txt for mycroft-volume
2021-10-25 17:35:55.580 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-volume
2021-10-25 17:35:55.580 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:35:55.582 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-volume.mycroftai
2021-10-25 17:35:55.594 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-volume.mycroftai/settings.json
2021-10-25 17:35:55.718 | ERROR    |  5713 | VolumeSkill | Couldn't allocate mixer, ALSAAudioError('No such file or directory [default]')
2021-10-25 17:35:58.729 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-volume.mycroftai
2021-10-25 17:35:58.730 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-volume.mycroftai/settings.json
2021-10-25 17:35:58.731 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-volume.mycroftai loaded successfully
2021-10-25 17:35:58.731 | INFO     |  5713 | __main__:on_started:177 | Skills service is starting up.
2021-10-25 17:35:58.803 | INFO     |  5713 | __main__:_update_system:154 | Attempting system update...
2021-10-25 17:36:01.806 | INFO     |  5713 | mycroft.enclosure.display_manager:_write_data:70 | Display Manager is creating /tmp/mycroft/ipc/managers/disp_info
2021-10-25 17:36:01.807 | INFO     |  5713 | __main__:_ensure_device_is_paired:133 | Device not paired, invoking the pairing skill
2021-10-25 17:36:01.809 | INFO     |  5713 | __main__:on_alive:181 | Skills service is alive.
2021-10-25 17:36:01.811 | INFO     |  5713 | msm.mycroft_skills_manager | building SkillEntry objects for all skills
2021-10-25 17:36:02.328 | INFO     |  5713 | PairingSkill | Communicating account URL to user
2021-10-25 17:36:02.899 | WARNING  |  5713 | msm.mycroft_skills_manager | No such default skill: mycroft-pandora
2021-10-25 17:36:02.900 | INFO     |  5713 | mycroft.skills.skill_manager:run:234 | Not all default skills are installed, performing skill update...
2021-10-25 17:36:02.900 | INFO     |  5713 | mycroft.skills.skill_updater:update_skills:149 | Beginning skill update...
2021-10-25 17:36:02.969 | INFO     |  5713 | msm.mycroft_skills_manager | building SkillEntry objects for all skills
2021-10-25 17:36:03.807 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:03.895 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-alarm
2021-10-25 17:36:03.898 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/fallback-unknown
2021-10-25 17:36:03.904 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-audio-record
2021-10-25 17:36:03.909 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-configuration
2021-10-25 17:36:03.909 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/fallback-wolfram-alpha
2021-10-25 17:36:03.910 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/fallback-duckduckgo
2021-10-25 17:36:03.910 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-hello-world
2021-10-25 17:36:03.929 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-date-time
2021-10-25 17:36:03.928 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-installer
2021-10-25 17:36:03.937 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-ip
2021-10-25 17:36:03.956 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-naptime
2021-10-25 17:36:03.957 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-npr-news
2021-10-25 17:36:04.003 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-personal
2021-10-25 17:36:03.992 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-joke
2021-10-25 17:36:04.011 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-singing
2021-10-25 17:36:04.064 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-playback-control
2021-10-25 17:36:04.159 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-reminder
2021-10-25 17:36:05.159 | ERROR    |  5713 | msm.mycroft_skills_manager | Error running install_or_update on homeassistant.mycroftai: MergeConflict
2021-10-25 17:36:05.164 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-speak
2021-10-25 17:36:05.925 | INFO     |  5713 | msm.skill_entry | Nothing new for mycroft-volume
2021-10-25 17:36:05.934 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:05.949 | INFO     |  5713 | msm.skill_entry | Installing requirements.txt for mycroft-volume
2021-10-25 17:36:06.033 | INFO     |  5713 | msm.skill_entry | Nothing new for mycroft-pairing
2021-10-25 17:36:06.040 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:06.095 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-spelling
2021-10-25 17:36:06.581 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:06.655 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:06.818 | INFO     |  5713 | msm.skill_entry | Installing requirements.txt for mycroft-audio-record
2021-10-25 17:36:06.824 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:06.829 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:06.906 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:06.912 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:06.915 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:06.961 | INFO     |  5713 | msm.skill_entry | Successfully installed fallback-unknown
2021-10-25 17:36:06.962 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:06.964 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-stop
2021-10-25 17:36:07.025 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:07.106 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-hello-world
2021-10-25 17:36:07.106 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:07.108 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-support
2021-10-25 17:36:07.109 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-naptime
2021-10-25 17:36:07.109 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:07.109 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/mycroft-timer
2021-10-25 17:36:07.115 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:07.177 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:07.193 | INFO     |  5713 | msm.skill_entry | Installing requirements.txt for mycroft-ip
2021-10-25 17:36:07.194 | INFO     |  5713 | msm.skill_entry | Installing requirements.txt for mycroft-joke
2021-10-25 17:36:07.219 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:07.252 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:07.272 | INFO     |  5713 | msm.skill_entry | Installing requirements.txt for mycroft-fallback-duck-duck-go
2021-10-25 17:36:07.275 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-playback-control
2021-10-25 17:36:07.275 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:07.277 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-weather
2021-10-25 17:36:07.288 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:07.318 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:07.369 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-personal
2021-10-25 17:36:07.369 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:07.370 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-version-checker
2021-10-25 17:36:07.376 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:07.400 | INFO     |  5713 | msm.skill_entry | Installing requirements.txt for fallback-wolfram-alpha
2021-10-25 17:36:07.471 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-speak
2021-10-25 17:36:07.471 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:07.473 | INFO     |  5713 | msm.skill_entry | Installing requirements.txt for mycroft-configuration
2021-10-25 17:36:07.475 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/mycroftai/skill-wiki
2021-10-25 17:36:07.505 | INFO     |  5713 | msm.skill_entry | Installing requirements.txt for mycroft-npr-news
2021-10-25 17:36:07.509 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:07.520 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-reminder
2021-10-25 17:36:07.521 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:07.542 | INFO     |  5713 | msm.skill_entry | Downloading skill: https://github.com/MycroftAI/skill-query
2021-10-25 17:36:07.570 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-spelling
2021-10-25 17:36:07.571 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:07.760 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:07.769 | INFO     |  5713 | msm.skill_entry | Installing requirements.txt for mycroft-installer
2021-10-25 17:36:07.868 | INFO     |  5713 | msm.skill_entry | Installing requirements.txt for mycroft-date-time
2021-10-25 17:36:08.085 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:08.147 | INFO     |  5713 | msm.skill_entry | Installing requirements.txt for mycroft-alarm
2021-10-25 17:36:08.272 | ERROR    |  5713 | msm.mycroft_skills_manager | Error running install_or_update on homeassistant: MergeConflict
2021-10-25 17:36:08.449 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:08.470 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:08.508 | INFO     |  5713 | msm.skill_entry | Installing requirements.txt for mycroft-support-helper
2021-10-25 17:36:08.558 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-stop
2021-10-25 17:36:08.559 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:08.625 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:08.673 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-version-checker
2021-10-25 17:36:08.674 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:09.068 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:09.101 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:09.141 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-timer
2021-10-25 17:36:09.146 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:09.174 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:09.183 | INFO     |  5713 | msm.skill_entry | Successfully installed fallback-query
2021-10-25 17:36:09.184 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:09.205 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:09.239 | INFO     |  5713 | msm.skill_entry | Installing requirements.txt for mycroft-weather
2021-10-25 17:36:09.250 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-singing
2021-10-25 17:36:09.250 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:09.251 | INFO     |  5713 | msm.skill_entry | Installing system requirements...
2021-10-25 17:36:09.272 | INFO     |  5713 | msm.skill_entry | Installing requirements.txt for mycroft-wiki
2021-10-25 17:36:10.514 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-audio-record
2021-10-25 17:36:10.515 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:11.444 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-ip
2021-10-25 17:36:11.444 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:12.596 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-joke
2021-10-25 17:36:12.597 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:13.705 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-fallback-duck-duck-go
2021-10-25 17:36:13.705 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:18.297 | INFO     |  5713 | msm.skill_entry | Successfully installed fallback-wolfram-alpha
2021-10-25 17:36:18.297 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:21.095 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-configuration
2021-10-25 17:36:21.095 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:24.531 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-npr-news
2021-10-25 17:36:24.532 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:25.567 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-installer
2021-10-25 17:36:25.567 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:32.740 | INFO     |  5713 | PairingSkill | Initiating device pairing sequence...
2021-10-25 17:36:32.741 | INFO     |  5713 | PairingSkill | Retrieving pairing code from device API...
2021-10-25 17:36:33.097 | INFO     |  5713 | PairingSkill | Pairing code obtained: AHEUC7
2021-10-25 17:36:33.097 | INFO     |  5713 | PairingSkill | Communicating pairing URL to user
2021-10-25 17:36:43.228 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-date-time
2021-10-25 17:36:43.229 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:44.222 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-alarm
2021-10-25 17:36:44.222 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:46.914 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-support-helper
2021-10-25 17:36:46.914 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:48.214 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-weather
2021-10-25 17:36:48.216 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:50.252 | INFO     |  5713 | msm.skill_entry | Successfully installed mycroft-wiki
2021-10-25 17:36:50.252 | INFO     |  5713 | msm.mycroft_skills_manager | invalidating skills cache
2021-10-25 17:36:50.338 | INFO     |  5713 | mycroft.skills.skill_updater:_log_next_download_time:267 | Next scheduled skill update: 2021-10-25 18:36:50.338105
2021-10-25 17:36:50.339 | INFO     |  5713 | mycroft.skills.skill_updater:update_skills:170 | Skill update complete
2021-10-25 17:36:50.339 | INFO     |  5713 | mycroft.skills.skill_manager:_load_on_startup:273 | Loading installed skills...
2021-10-25 17:36:50.341 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-ip.mycroftai
2021-10-25 17:36:50.394 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-ip.mycroftai/settings.json
2021-10-25 17:36:50.407 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-ip.mycroftai
2021-10-25 17:36:50.408 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-ip.mycroftai/settings.json
2021-10-25 17:36:50.409 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-ip.mycroftai loaded successfully
2021-10-25 17:36:50.410 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-reminder.mycroftai
2021-10-25 17:36:50.453 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-reminder.mycroftai/settings.json
2021-10-25 17:36:50.507 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-reminder.mycroftai
2021-10-25 17:36:50.511 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-reminder.mycroftai/settings.json
2021-10-25 17:36:50.511 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-reminder.mycroftai loaded successfully
2021-10-25 17:36:50.512 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-installer.mycroftai
2021-10-25 17:36:50.521 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-installer.mycroftai/settings.json
2021-10-25 17:36:50.550 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-installer.mycroftai
2021-10-25 17:36:50.551 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-installer.mycroftai/settings.json
2021-10-25 17:36:50.552 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-installer.mycroftai loaded successfully
2021-10-25 17:36:50.552 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-version-checker.mycroftai
2021-10-25 17:36:50.561 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-version-checker.mycroftai/settings.json
2021-10-25 17:36:50.583 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-version-checker.mycroftai
2021-10-25 17:36:50.586 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-version-checker.mycroftai/settings.json
2021-10-25 17:36:50.589 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-version-checker.mycroftai loaded successfully
2021-10-25 17:36:50.590 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-singing.mycroftai
2021-10-25 17:36:50.599 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-singing.mycroftai/settings.json
2021-10-25 17:36:50.613 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-singing.mycroftai
2021-10-25 17:36:50.615 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-singing.mycroftai/settings.json
2021-10-25 17:36:50.618 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-singing.mycroftai loaded successfully
2021-10-25 17:36:50.619 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-joke.mycroftai
2021-10-25 17:36:50.627 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-joke.mycroftai/settings.json
2021-10-25 17:36:50.647 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-joke.mycroftai
2021-10-25 17:36:50.649 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-joke.mycroftai/settings.json
2021-10-25 17:36:50.651 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-joke.mycroftai loaded successfully
2021-10-25 17:36:50.651 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-timer.mycroftai
2021-10-25 17:36:50.672 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-timer.mycroftai/settings.json
2021-10-25 17:36:50.743 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-timer.mycroftai
2021-10-25 17:36:50.746 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-timer.mycroftai/settings.json
2021-10-25 17:36:50.749 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-timer.mycroftai loaded successfully
2021-10-25 17:36:50.750 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: homeassistant.mycroftai
2021-10-25 17:36:51.103 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/homeassistant.mycroftai/settings.json
2021-10-25 17:36:51.104 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/homeassistant.mycroftai/settings.json
2021-10-25 17:36:51.193 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill homeassistant.mycroftai loaded successfully
2021-10-25 17:36:51.193 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-alarm.mycroftai
2021-10-25 17:36:51.226 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-alarm.mycroftai/settings.json
2021-10-25 17:36:51.226 | ERROR    |  5713 | AlarmSkill | Couldn't allocate mixer, ALSAAudioError('No such file or directory [default]')
2021-10-25 17:36:51.311 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-alarm.mycroftai
2021-10-25 17:36:51.324 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-alarm.mycroftai/settings.json
2021-10-25 17:36:51.326 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-alarm.mycroftai loaded successfully
2021-10-25 17:36:51.326 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-naptime.mycroftai
2021-10-25 17:36:51.330 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-naptime.mycroftai/settings.json
2021-10-25 17:36:51.353 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-naptime.mycroftai
2021-10-25 17:36:51.354 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-naptime.mycroftai/settings.json
2021-10-25 17:36:51.360 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-naptime.mycroftai loaded successfully
2021-10-25 17:36:51.361 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-spelling.mycroftai
2021-10-25 17:36:51.364 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-spelling.mycroftai/settings.json
2021-10-25 17:36:51.388 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-spelling.mycroftai
2021-10-25 17:36:51.393 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-spelling.mycroftai/settings.json
2021-10-25 17:36:51.396 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-spelling.mycroftai loaded successfully
2021-10-25 17:36:51.397 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-fallback-duck-duck-go.mycroftai
2021-10-25 17:36:51.412 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-fallback-duck-duck-go.mycroftai/settings.json
2021-10-25 17:36:51.450 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-fallback-duck-duck-go.mycroftai
2021-10-25 17:36:51.451 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-fallback-duck-duck-go.mycroftai/settings.json
2021-10-25 17:36:51.456 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-fallback-duck-duck-go.mycroftai loaded successfully
2021-10-25 17:36:51.457 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-npr-news.mycroftai
2021-10-25 17:36:51.584 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-npr-news.mycroftai/settings.json
2021-10-25 17:36:52.621 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-npr-news.mycroftai
2021-10-25 17:36:52.623 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-npr-news.mycroftai/settings.json
2021-10-25 17:36:52.624 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-npr-news.mycroftai loaded successfully
2021-10-25 17:36:52.624 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-support-helper.mycroftai
2021-10-25 17:36:52.632 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-support-helper.mycroftai/settings.json
2021-10-25 17:36:52.639 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-support-helper.mycroftai
2021-10-25 17:36:52.641 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-support-helper.mycroftai/settings.json
2021-10-25 17:36:52.642 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-support-helper.mycroftai loaded successfully
2021-10-25 17:36:52.643 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-weather.mycroftai
2021-10-25 17:36:52.666 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-weather.mycroftai/settings.json
2021-10-25 17:36:52.819 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-weather.mycroftai
2021-10-25 17:36:52.820 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-weather.mycroftai/settings.json
2021-10-25 17:36:52.823 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-weather.mycroftai loaded successfully
2021-10-25 17:36:52.823 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-speak.mycroftai
2021-10-25 17:36:52.834 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-speak.mycroftai/settings.json
2021-10-25 17:36:52.954 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-speak.mycroftai
2021-10-25 17:36:52.955 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-speak.mycroftai/settings.json
2021-10-25 17:36:53.034 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-speak.mycroftai loaded successfully
2021-10-25 17:36:53.035 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-audio-record.mycroftai
2021-10-25 17:36:53.048 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-audio-record.mycroftai/settings.json
2021-10-25 17:36:53.081 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-audio-record.mycroftai
2021-10-25 17:36:53.082 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-audio-record.mycroftai/settings.json
2021-10-25 17:36:53.100 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-audio-record.mycroftai loaded successfully
2021-10-25 17:36:53.101 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-stop.mycroftai
2021-10-25 17:36:53.105 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-stop.mycroftai/settings.json
2021-10-25 17:36:53.132 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-stop.mycroftai
2021-10-25 17:36:53.136 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-stop.mycroftai/settings.json
2021-10-25 17:36:53.161 | INFO     |  5713 | mycroft.skills.skill_loader:_communicate_load_status:344 | Skill mycroft-stop.mycroftai loaded successfully
2021-10-25 17:36:53.162 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-hello-world.mycroftai
2021-10-25 17:36:53.166 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-hello-world.mycroftai/settings.json
2021-10-25 17:36:53.182 | INFO     |  5713 | mycroft.skills.skill_loader:_check_for_first_run:324 | First run of mycroft-hello-world.mycroftai
2021-10-25 17:36:53.183 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-hello-world.mycroftai/settings.json
2021-10-25 17:36:53.202 | ERROR    |  5713 | mycroft_bus_client.client.client | === ConnectionResetError(104, 'Connection reset by peer') ===
Traceback (most recent call last):
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 305, in run_forever
    dispatcher.read(self.sock.sock, read, check)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 53, in read
    if not read_callback():
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 271, in read
    op_code, frame = self.sock.recv_data_frame(True)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 340, in recv_data_frame
    frame = self.recv_frame()
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 374, in recv_frame
    return self.frame_buffer.recv_frame()
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_abnf.py", line 361, in recv_frame
    self.recv_header()
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_abnf.py", line 309, in recv_header
    header = self.recv_strict(2)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_abnf.py", line 396, in recv_strict
    bytes_ = self.recv(min(16384, shortage))
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 449, in _recv
    return recv(self.sock, bufsize)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_socket.py", line 81, in recv
    bytes_ = sock.recv(bufsize)
ConnectionResetError: [Errno 104] Connection reset by peer
2021-10-25 17:36:53.210 | ERROR    |  5713 | mycroft_bus_client.client.client | Exception closing websocket: ConnectionResetError(104, 'Connection reset by peer')
2021-10-25 17:36:53.210 | WARNING  |  5713 | mycroft_bus_client.client.client | Message Bus Client will reconnect in 5 seconds.
2021-10-25 17:36:53.208 | ERROR    |  5713 | mycroft.skills.skill_manager:_load_skill:305 | Load of skill /opt/mycroft/skills/mycroft-hello-world.mycroftai failed!
Traceback (most recent call last):
  File "/opt/mycroft-core/mycroft/skills/skill_manager.py", line 303, in _load_skill
    load_status = skill_loader.load()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 186, in load
    return self._load()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 249, in _load
    self._communicate_load_status()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 343, in _communicate_load_status
    self.bus.emit(message)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/mycroft_bus_client/client/client.py", line 160, in emit
    self.client.send(message.serialize())
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 153, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 250, in send
    return self.send_frame(frame)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 275, in send_frame
    l = self._send(data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 445, in _send
    return send(self.sock, data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_socket.py", line 117, in send
    return sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
2021-10-25 17:36:53.211 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-date-time.mycroftai
2021-10-25 17:36:53.704 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-date-time.mycroftai/settings.json
2021-10-25 17:36:53.710 | ERROR    |  5713 | mycroft.skills.skill_manager:_load_skill:305 | Load of skill /opt/mycroft/skills/mycroft-date-time.mycroftai failed!
Traceback (most recent call last):
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 302, in _create_skill_instance
    self.instance.load_data_files()
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1274, in load_data_files
    self.load_vocab_files(root_directory)
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1300, in load_vocab_files
    aliases)
  File "/opt/mycroft-core/mycroft/skills/intent_service_interface.py", line 56, in register_adapt_keyword
    {**entity_data, **compatibility_data})
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/mycroft_bus_client/client/client.py", line 160, in emit
    self.client.send(message.serialize())
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 153, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 250, in send
    return self.send_frame(frame)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 275, in send_frame
    l = self._send(data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 445, in _send
    return send(self.sock, data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_socket.py", line 117, in send
    return sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/mycroft-core/mycroft/skills/skill_manager.py", line 303, in _load_skill
    load_status = skill_loader.load()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 186, in load
    return self._load()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 244, in _load
    if skill_module and self._create_skill_instance(skill_module):
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 310, in _create_skill_instance
    self.instance.default_shutdown()
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1374, in default_shutdown
    self.gui.shutdown()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 376, in shutdown
    self.release()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 366, in release
    self.clear()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 130, in clear
    {"__from": self.skill.skill_id}))
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/mycroft_bus_client/client/client.py", line 160, in emit
    self.client.send(message.serialize())
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 153, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 250, in send
    return self.send_frame(frame)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 275, in send_frame
    l = self._send(data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 445, in _send
    return send(self.sock, data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_socket.py", line 117, in send
    return sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
2021-10-25 17:36:53.710 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-configuration.mycroftai
2021-10-25 17:36:53.716 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-configuration.mycroftai/settings.json
2021-10-25 17:36:53.723 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-configuration.mycroftai/settings.json
2021-10-25 17:36:53.724 | ERROR    |  5713 | mycroft.skills.skill_manager:_load_skill:305 | Load of skill /opt/mycroft/skills/mycroft-configuration.mycroftai failed!
Traceback (most recent call last):
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 302, in _create_skill_instance
    self.instance.load_data_files()
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1274, in load_data_files
    self.load_vocab_files(root_directory)
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1300, in load_vocab_files
    aliases)
  File "/opt/mycroft-core/mycroft/skills/intent_service_interface.py", line 56, in register_adapt_keyword
    {**entity_data, **compatibility_data})
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/mycroft_bus_client/client/client.py", line 160, in emit
    self.client.send(message.serialize())
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 153, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 250, in send
    return self.send_frame(frame)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 275, in send_frame
    l = self._send(data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 445, in _send
    return send(self.sock, data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_socket.py", line 117, in send
    return sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/mycroft-core/mycroft/skills/skill_manager.py", line 303, in _load_skill
    load_status = skill_loader.load()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 186, in load
    return self._load()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 244, in _load
    if skill_module and self._create_skill_instance(skill_module):
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 310, in _create_skill_instance
    self.instance.default_shutdown()
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1374, in default_shutdown
    self.gui.shutdown()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 376, in shutdown
    self.release()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 366, in release
    self.clear()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 130, in clear
    {"__from": self.skill.skill_id}))
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/mycroft_bus_client/client/client.py", line 160, in emit
    self.client.send(message.serialize())
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 153, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 250, in send
    return self.send_frame(frame)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 275, in send_frame
    l = self._send(data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 445, in _send
    return send(self.sock, data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_socket.py", line 117, in send
    return sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
2021-10-25 17:36:53.724 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: fallback-query.mycroftai
2021-10-25 17:36:53.729 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/fallback-query.mycroftai/settings.json
2021-10-25 17:36:53.733 | INFO     |  5713 | QuestionsAnswersSkill | Removing all handlers...
2021-10-25 17:36:53.733 | ERROR    |  5713 | mycroft.skills.skill_manager:_load_skill:305 | Load of skill /opt/mycroft/skills/fallback-query.mycroftai failed!
Traceback (most recent call last):
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 302, in _create_skill_instance
    self.instance.load_data_files()
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1274, in load_data_files
    self.load_vocab_files(root_directory)
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1300, in load_vocab_files
    aliases)
  File "/opt/mycroft-core/mycroft/skills/intent_service_interface.py", line 56, in register_adapt_keyword
    {**entity_data, **compatibility_data})
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/mycroft_bus_client/client/client.py", line 160, in emit
    self.client.send(message.serialize())
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 153, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 250, in send
    return self.send_frame(frame)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 275, in send_frame
    l = self._send(data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 445, in _send
    return send(self.sock, data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_socket.py", line 117, in send
    return sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/mycroft-core/mycroft/skills/skill_manager.py", line 303, in _load_skill
    load_status = skill_loader.load()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 186, in load
    return self._load()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 244, in _load
    if skill_module and self._create_skill_instance(skill_module):
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 310, in _create_skill_instance
    self.instance.default_shutdown()
  File "/opt/mycroft-core/mycroft/skills/fallback_skill.py", line 199, in default_shutdown
    super(FallbackSkill, self).default_shutdown()
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1374, in default_shutdown
    self.gui.shutdown()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 376, in shutdown
    self.release()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 366, in release
    self.clear()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 130, in clear
    {"__from": self.skill.skill_id}))
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/mycroft_bus_client/client/client.py", line 160, in emit
    self.client.send(message.serialize())
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 153, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 250, in send
    return self.send_frame(frame)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 275, in send_frame
    l = self._send(data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 445, in _send
    return send(self.sock, data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_socket.py", line 117, in send
    return sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
2021-10-25 17:36:53.734 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-personal.mycroftai
2021-10-25 17:36:53.738 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-personal.mycroftai/settings.json
2021-10-25 17:36:53.742 | ERROR    |  5713 | mycroft.skills.skill_manager:_load_skill:305 | Load of skill /opt/mycroft/skills/mycroft-personal.mycroftai failed!
Traceback (most recent call last):
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 305, in _create_skill_instance
    self.instance._register_decorated()
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 749, in _register_decorated
    self.register_intent(intent, method)
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 985, in register_intent
    return self.register_intent_file(intent_parser, handler)
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1022, in register_intent_file
    self.intent_service.register_padatious_intent(name, filename)
  File "/opt/mycroft-core/mycroft/skills/intent_service_interface.py", line 129, in register_padatious_intent
    self.bus.emit(Message("padatious:register_intent", data))
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/mycroft_bus_client/client/client.py", line 160, in emit
    self.client.send(message.serialize())
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 153, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 250, in send
    return self.send_frame(frame)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 275, in send_frame
    l = self._send(data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 445, in _send
    return send(self.sock, data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_socket.py", line 117, in send
    return sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/mycroft-core/mycroft/skills/skill_manager.py", line 303, in _load_skill
    load_status = skill_loader.load()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 186, in load
    return self._load()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 244, in _load
    if skill_module and self._create_skill_instance(skill_module):
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 310, in _create_skill_instance
    self.instance.default_shutdown()
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1374, in default_shutdown
    self.gui.shutdown()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 376, in shutdown
    self.release()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 366, in release
    self.clear()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 130, in clear
    {"__from": self.skill.skill_id}))
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/mycroft_bus_client/client/client.py", line 160, in emit
    self.client.send(message.serialize())
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 153, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 250, in send
    return self.send_frame(frame)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 275, in send_frame
    l = self._send(data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 445, in _send
    return send(self.sock, data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_socket.py", line 117, in send
    return sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
2021-10-25 17:36:53.743 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: fallback-wolfram-alpha.mycroftai
2021-10-25 17:36:53.768 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/fallback-wolfram-alpha.mycroftai/settings.json
2021-10-25 17:36:53.776 | ERROR    |  5713 | mycroft.skills.skill_manager:_load_skill:305 | Load of skill /opt/mycroft/skills/fallback-wolfram-alpha.mycroftai failed!
Traceback (most recent call last):
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 302, in _create_skill_instance
    self.instance.load_data_files()
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1274, in load_data_files
    self.load_vocab_files(root_directory)
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1300, in load_vocab_files
    aliases)
  File "/opt/mycroft-core/mycroft/skills/intent_service_interface.py", line 56, in register_adapt_keyword
    {**entity_data, **compatibility_data})
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/mycroft_bus_client/client/client.py", line 160, in emit
    self.client.send(message.serialize())
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 153, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 250, in send
    return self.send_frame(frame)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 275, in send_frame
    l = self._send(data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 445, in _send
    return send(self.sock, data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_socket.py", line 117, in send
    return sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/mycroft-core/mycroft/skills/skill_manager.py", line 303, in _load_skill
    load_status = skill_loader.load()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 186, in load
    return self._load()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 244, in _load
    if skill_module and self._create_skill_instance(skill_module):
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 310, in _create_skill_instance
    self.instance.default_shutdown()
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1374, in default_shutdown
    self.gui.shutdown()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 376, in shutdown
    self.release()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 366, in release
    self.clear()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 130, in clear
    {"__from": self.skill.skill_id}))
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/mycroft_bus_client/client/client.py", line 160, in emit
    self.client.send(message.serialize())
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 153, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 250, in send
    return self.send_frame(frame)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 275, in send_frame
    l = self._send(data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 445, in _send
    return send(self.sock, data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_socket.py", line 117, in send
    return sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
2021-10-25 17:36:53.777 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-wiki.mycroftai
2021-10-25 17:36:53.782 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-wiki.mycroftai/settings.json
2021-10-25 17:36:53.787 | ERROR    |  5713 | mycroft.skills.skill_manager:_load_skill:305 | Load of skill /opt/mycroft/skills/mycroft-wiki.mycroftai failed!
Traceback (most recent call last):
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 302, in _create_skill_instance
    self.instance.load_data_files()
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1274, in load_data_files
    self.load_vocab_files(root_directory)
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1300, in load_vocab_files
    aliases)
  File "/opt/mycroft-core/mycroft/skills/intent_service_interface.py", line 56, in register_adapt_keyword
    {**entity_data, **compatibility_data})
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/mycroft_bus_client/client/client.py", line 160, in emit
    self.client.send(message.serialize())
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 153, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 250, in send
    return self.send_frame(frame)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 275, in send_frame
    l = self._send(data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 445, in _send
    return send(self.sock, data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_socket.py", line 117, in send
    return sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/mycroft-core/mycroft/skills/skill_manager.py", line 303, in _load_skill
    load_status = skill_loader.load()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 186, in load
    return self._load()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 244, in _load
    if skill_module and self._create_skill_instance(skill_module):
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 310, in _create_skill_instance
    self.instance.default_shutdown()
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1374, in default_shutdown
    self.gui.shutdown()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 376, in shutdown
    self.release()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 366, in release
    self.clear()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 130, in clear
    {"__from": self.skill.skill_id}))
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/mycroft_bus_client/client/client.py", line 160, in emit
    self.client.send(message.serialize())
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 153, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 250, in send
    return self.send_frame(frame)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 275, in send_frame
    l = self._send(data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 445, in _send
    return send(self.sock, data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_socket.py", line 117, in send
    return sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
2021-10-25 17:36:53.788 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: fallback-unknown.mycroftai
2021-10-25 17:36:53.791 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/fallback-unknown.mycroftai/settings.json
2021-10-25 17:36:53.796 | INFO     |  5713 | UnknownSkill | Removing all handlers...
2021-10-25 17:36:53.796 | ERROR    |  5713 | mycroft.skills.skill_manager:_load_skill:305 | Load of skill /opt/mycroft/skills/fallback-unknown.mycroftai failed!
Traceback (most recent call last):
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 302, in _create_skill_instance
    self.instance.load_data_files()
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1274, in load_data_files
    self.load_vocab_files(root_directory)
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1300, in load_vocab_files
    aliases)
  File "/opt/mycroft-core/mycroft/skills/intent_service_interface.py", line 56, in register_adapt_keyword
    {**entity_data, **compatibility_data})
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/mycroft_bus_client/client/client.py", line 160, in emit
    self.client.send(message.serialize())
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 153, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 250, in send
    return self.send_frame(frame)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 275, in send_frame
    l = self._send(data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 445, in _send
    return send(self.sock, data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_socket.py", line 117, in send
    return sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/mycroft-core/mycroft/skills/skill_manager.py", line 303, in _load_skill
    load_status = skill_loader.load()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 186, in load
    return self._load()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 244, in _load
    if skill_module and self._create_skill_instance(skill_module):
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 310, in _create_skill_instance
    self.instance.default_shutdown()
  File "/opt/mycroft-core/mycroft/skills/fallback_skill.py", line 199, in default_shutdown
    super(FallbackSkill, self).default_shutdown()
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1374, in default_shutdown
    self.gui.shutdown()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 376, in shutdown
    self.release()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 366, in release
    self.clear()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 130, in clear
    {"__from": self.skill.skill_id}))
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/mycroft_bus_client/client/client.py", line 160, in emit
    self.client.send(message.serialize())
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 153, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 250, in send
    return self.send_frame(frame)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 275, in send_frame
    l = self._send(data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 445, in _send
    return send(self.sock, data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_socket.py", line 117, in send
    return sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
2021-10-25 17:36:53.797 | INFO     |  5713 | mycroft.skills.skill_loader:load:185 | ATTEMPTING TO LOAD SKILL: mycroft-playback-control.mycroftai
2021-10-25 17:36:53.802 | INFO     |  5713 | mycroft.skills.settings:get_local_settings:83 | /opt/mycroft/skills/mycroft-playback-control.mycroftai/settings.json
2021-10-25 17:36:53.808 | ERROR    |  5713 | mycroft.skills.skill_manager:_load_skill:305 | Load of skill /opt/mycroft/skills/mycroft-playback-control.mycroftai failed!
Traceback (most recent call last):
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 302, in _create_skill_instance
    self.instance.load_data_files()
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1274, in load_data_files
    self.load_vocab_files(root_directory)
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1300, in load_vocab_files
    aliases)
  File "/opt/mycroft-core/mycroft/skills/intent_service_interface.py", line 56, in register_adapt_keyword
    {**entity_data, **compatibility_data})
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/mycroft_bus_client/client/client.py", line 160, in emit
    self.client.send(message.serialize())
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 153, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 250, in send
    return self.send_frame(frame)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 275, in send_frame
    l = self._send(data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 445, in _send
    return send(self.sock, data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_socket.py", line 117, in send
    return sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/mycroft-core/mycroft/skills/skill_manager.py", line 303, in _load_skill
    load_status = skill_loader.load()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 186, in load
    return self._load()
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 244, in _load
    if skill_module and self._create_skill_instance(skill_module):
  File "/opt/mycroft-core/mycroft/skills/skill_loader.py", line 310, in _create_skill_instance
    self.instance.default_shutdown()
  File "/opt/mycroft-core/mycroft/skills/mycroft_skill/mycroft_skill.py", line 1374, in default_shutdown
    self.gui.shutdown()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 376, in shutdown
    self.release()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 366, in release
    self.clear()
  File "/opt/mycroft-core/mycroft/enclosure/gui.py", line 130, in clear
    {"__from": self.skill.skill_id}))
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/mycroft_bus_client/client/client.py", line 160, in emit
    self.client.send(message.serialize())
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_app.py", line 153, in send
    if not self.sock or self.sock.send(data, opcode) == 0:
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 250, in send
    return self.send_frame(frame)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 275, in send_frame
    l = self._send(data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_core.py", line 445, in _send
    return send(self.sock, data)
  File "/opt/mycroft-core/.venv/lib/python3.7/site-packages/websocket/_socket.py", line 117, in send
    return sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
2021-10-25 17:36:53.808 | INFO     |  5713 | mycroft.skills.skill_manager:_load_on_startup:275 | Skills all loaded!
2021-10-25 17:36:58.214 | INFO     |  5713 | mycroft_bus_client.client.client | Connected
2021-10-25 17:37:03.433 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-pairing.mycroftai/settings.json
2021-10-25 17:37:03.734 | INFO     |  5713 | mycroft.skills.settings:save_settings:116 | Skill settings successfully saved to /home/runner/.config/mycroft/skills/mycroft-timer.mycroftai/settings.json
